brainiak/fcma/preprocessing.py

```diff
@@ -303,7 +397,7 @@ def prepare_searchlight_mvpa_data(images, conditions, data_type=np.float32):
                 processed_data[:, :, :, idx] = \
                     np.mean(data[:, :, :, epoch[2]:epoch[3]], axis=3)
 
-        logger.info(
+        logger.debug(
             'file %s is loaded and processed, with data shape %s' %
```

Comment:
logger does formatting itself, you can just do:
`logger.debug('file %s is loaded and processed, with data shape $s', f.get_filename(), data.shape)`